scripts/test.sh

```diff
@@ -46,7 +46,13 @@ start_ganache() {
 
   ganache_pid=$!
 
-  sleep 1
+  echo "Waiting for ganache to launch on port "$ganache_port"..."
+
+  while ! nc -z localhost $ganache_port; do
```

Comment:
I just realized we can reuse the `ganache_running` function defined earlier in this file.

```suggestion
  while ! ganache_running; do
```

Reply:
@frangio Oh, yes. You already had done the work half the way. Did not see the "Commit suggestion" option. So I force pushed your proposal into my branch. Anyway, that was easy :-)